resources/views/mycourses.blade.php

```diff
@@ -0,0 +1,132 @@
+@extends('layouts/main')
+@section('head')
+<script src="{{ asset('js/jquery-1.11.3.min.js')}}"></script>
```

Comment:
@saidabdul80 is there a reason we are using Jquery for this project when we have Vue already as part of it?

Reply:
Sorry I don't understand sir, jQuery is just like a code that hold library of function. So I don't see the effect of this library to the performance of the system, and however I didn't bring that to table at first because of my understanding of what this library is. But however sir it's not too late to include

Reply:
I thought you got welled review of vicelab. And however no special meeting was done for what and what is to use in general. I could remember it was just like questions

Reply:
So I was just wondering if we needed the library on the project.